x-pack/plugins/fleet/server/services/epm/packages/get.ts

```diff
@@ -68,13 +72,19 @@ export async function getPackages(
   });
   // get the installed packages
   const packageSavedObjects = await getPackageSavedObjects(savedObjectsClient);
+
+  const packagesNotInRegistry = packageSavedObjects.saved_objects
+    .filter((pkg) => !registryItems.some((item) => item.name === pkg.id))
+    .map((pkg) => createInstallableFrom({ ...pkg.attributes, title: nameAsTitle(pkg.id) }, pkg));
```

Comment:
Noticed that saved objects are missing `title`, `description`. So populating title as the capitalized name. E.g. 'my-package' -> 'My-package'

Reply:
Could we get these info from the package manifest? maybe it has a big performance impact not sure.

Reply:
Probably it's not a big impact as we would have only a few of these uploaded packages.

Reply:
Raised https://github.com/elastic/kibana/issues/150409